app/models/admin_ability.rb

```diff
@@ -120,6 +125,10 @@ def signed_in_with_organizer_role(user, conf_ids_for_organization_admin = [])
     can :manage, Registration, conference_id: conf_ids
     can :manage, RegistrationPeriod, conference_id: conf_ids
     can :manage, Booth, conference_id: conf_ids
+    cannot :accept, Booth do |booth|
```

Comment:
That's right. And moreover it is about the order that things are defined. Even if there was no admin and no admin abilities, common_abilities call the other functions for the abilities for organizer, cfp role etc. If you wanted something (the same thing) to apply for all roles, you could put it in one place, as long as it was **after** their abilities definition, ie at the end of common_abilities.